### PR TITLE
minecraft service's UI added

### PR DESCRIPTION
**Pull request by DaddyWesker**
I have added read-to-be-used UI for new Minecraftizing service.

### CODE REVIEW

**Comment by anandrgitnirman** [comment]
@DaddyWesker ,you will need to Raise the PR against the development branch . 
Deployment to mainnet will be done from Development branch 
@Vivek205 willl come back with any code review comments

**Comment by DaddyWesker** [comment]
@anandrgitnirman 
Got it. Changed to "development"

File: src/assets/thirdPartyServices/snet/minecraftizing_service/index.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          </div>
+        </div>
+        <div className="row" align="center">
+          <SNETImageUpload imageName={""} imageDataFunc={this.getImageData} instantUrlFetch={true} allowURL={true} />
+        </div>
+        <div className="row">
+          <div className="col-md-3 col-lg-3" style={{ padding: "10px", fontSize: "13px", marginLeft: "10px" }}>
+            About:{" "}
+          </div>
+          <div className="col-xs-3 col-xs-2">
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/index.js]: this div is repeating. extract it as a separate component. The data could be kept in an array and the same could be iterated using the map function.

File: src/assets/thirdPartyServices/snet/minecraftizing_service/index.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { Grid, IconButton, MuiThemeProvider, Tooltip } from "@material-ui/core";
+
+import { MinecraftizingService } from "./MinecraftizingService_pb_service";
+import RandomDropDown from "./DropDown";
+import MethodNamesDropDown from "../../common/MethodNamesDropDown";
+import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+
+// const outsideWrapper = {
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/index.js]: remove the comments if not required

File: src/assets/thirdPartyServices/snet/minecraftizing_service/index.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+  renderForm() {
+    const serviceNameOptions = ["Select a method", ...this.props.serviceClient.getMethodNames(MinecraftizingService)];
+    const modelNames = ["Select model", "UGATIT", "cycle_gan"];
+    const datasetNames = ["Select dataset", "minecraft_landscapes"];
+
+    return (
+      <React.Fragment>
+        <div className="row">
+          <div className="col-md-3 col-lg-3" style={{ padding: "10px", fontSize: "13px", marginLeft: "10px" }}>
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/index.js]: avoid inline styling.

File: src/assets/thirdPartyServices/snet/minecraftizing_service/DropDown.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const RandomDropDown = ({ name, list, value, onChange, ...restProps }) => {
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/DropDown.js]: Use StyledDropdown component in the `src/components/common/`

File: src/assets/thirdPartyServices/snet/minecraftizing_service/index.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+            Method Name:{" "}
+          </div>
+          <div className="col-md-3 col-lg-3">
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/index.js]: we're not using bootstrap anymore. please get rid of the bootstrap classNames. e.g `className="col-md-3 col-lg-3"`

**Review by Vivek205 — COMMENTED** [review]
Suggested some improvements. Please have a look.

**Comment by DaddyWesker** [comment]
I've updated code in index.js a bit. Please consider additional code review, @Vivek205

File: src/assets/thirdPartyServices/snet/minecraftizing_service/index.js [hunk 6]
```diff
@@ -91,76 +78,70 @@ export default class MinecraftService extends React.Component {
...
+
+  repeatedButton(hrf, data)
+  {
+      return (
+      <Button target="_blank" href={hrf} style={{ fontSize: "13px", marginLeft: "10px" }}>
+          <ul>{data}</ul>
+      </Button>
+      );
+  }
+
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/snet/minecraftizing_service/index.js]: modularize these methods as separate stateless-components.

**Review by Vivek205 — COMMENTED** [review]
thanks for the efforts..
went through the changes in index.js and updated the comments.
could you have a look at this https://github.com/singnet/snet-dapp/pull/395#discussion_r330010727 as well

**Comment by DaddyWesker** [comment]
@Vivek205 
Hm. Cant see updated comments, only old ones. Where to find them?
Also, about "modularize these methods as separate stateless-components" you mean to extract those methods from the class?
upd. Found styleddropdown location but now i need to change too much. But if it's so necessary...

**Comment by DaddyWesker** [comment]
I dont get this StyledDropdown. I've added this, now checking how it works. It works really strange. It shows no default image just empty field with down arrow. Like this. 
image

Though it has "Select a value" as a default value which is weird. 
image

**Comment by DaddyWesker** [comment]
Alright... i've updated once again index.js file. Can you consider yet another look at it, @Vivek205 ?

**Review by Vivek205 — APPROVED** [review]
:+1:

**Comment by DaddyWesker** [comment]
Anything needs to be done yet to be pushed to development branch, @Vivek205 ? Since it is still there... in pull requests

**Pull request merged**